libs/langchain/langchain/chat_models/vertexai.py

```diff
@@ -177,16 +174,22 @@ def _generate(
 
         question = _get_question(messages)
         history = _parse_chat_history(messages[:-1])
-        params = self._prepare_params(stop=stop, **kwargs)
+        params = self._prepare_params(stop=stop, stream=should_stream, **kwargs)
```

Comment:
if should_stream is true we would have already returned in L173

Reply:
agree, fixed.